packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx

```diff
@@ -122,10 +122,9 @@ const TranslationInput = (props: TranslationInputProps) => {
     if (onFocus) onFocus();
   }, [props.isCollapsed, onFocus, onToggle]);
 
-  // This checks if the content in the textarea overflows the minimum
-  // amount of lines it should have when collapsed
-  const contentExceedsShownRows =
-    contentRowCount > TranslationInput.MIN_ROW_COUNT;
+  // This checks if the content in the textarea is greater than 38 (one row). If it is, then the toggle button will be shown.
+  // This is to prevent the toggle button from showing when there is not enough content to expand/collapse.
+  // const contentExceedsShownRows = contentRowCount > firstRowHeight;
```

Comment:
I think we should remove this comment block.

Reply:
Yes, definitely. On it!

Reply:
Removed here - [3a6cb5f](https://github.com/commercetools/ui-kit/pull/2511/commits/3a6cb5fab6d0fbb5880a1d41d4fe5b24024c2f78).